### PR TITLE
Added Contributing.md

### PR DESCRIPTION
**Pull request by nimit2801**
- Added the contributing.md for [Hackathons](https://github.com/hackclub/hackathons)
- With some cool gifs and informative images
- Solving #37 
- Please review thank you.

### CODE REVIEW

**Review by sampoder — CHANGES_REQUESTED** [review]
Thanks for contributing :)

Unfortunately, I don't think this is up to standard. A few pointers:

- Headings should only be used for headings not body text
- The grammar needs significant improving 
- The "vibe" of the text feels off, it feels unprofessional but not in a good way

Hope you can understand

**Comment by nimit2801** [comment]
@sampoder I've made the changes requested.
Thank you for the review.

**Review by sampoder — CHANGES_REQUESTED** [review]
It's better, but there are still a lot of grammar errors

**Comment by nimit2801** [comment]
> 
> 
> It's better, but there are still a lot of grammar errors

Can you tell me which lines I should, improve or some suggestions.
Thank You.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
-# Contribution Guildlines
+# Contribution Guidelines
 
-#### This document contains a set of guidelines which any contributions to any Hack Club repository must follow.
+#### This document contains a set of guidelines that any contributions to any Hack Club repository must follow.
```
Comment (sampoder) [inline CONTRIBUTING.md]: G

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
 
-## Welcome Hackers !
+## Welcome Hackers!
 
 ### Step 1: Fork repo.
 
-#### Fork the original repo [hackathons](https://github.com/hackclub/hackathons) !
+Fork the original repo [hackathons](https://github.com/hackclub/hackathons) !
```
Comment (sampoder) [inline CONTRIBUTING.md:9]: G

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
 
 ![Fork](https://cloud-99tamsbv5-hack-club-bot.vercel.app/0fork.png)
 
-#### After this step you'll get your own copy of the repo.
+After this step, you'll get your copy of the repo.
 
 ![forked](https://cloud-99tamsbv5-hack-club-bot.vercel.app/1forked.png)
 
-#### Something in your URL would look like:
+Something in your URL would look like this:
```
Comment (sampoder) [inline CONTRIBUTING.md:17]: G

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -38,9 +38,9 @@ cd hackathons
 git checkout -b <name-of-your-branch>
 ```
 
-### Step 4: Make your Awsome changes!
+### Step 4: Make your Awesome changes!
 
-![Awsome changes](https://cloud-qoynsmfj3-hack-club-bot.vercel.app/0tenor.gif)
+eg: In my case, I was adding the CONTRIBUTING.md
```
Comment (sampoder) [inline CONTRIBUTING.md:43]: G

**Comment by sampoder** [comment]
View above comments. Each has a grammar / phrasing issue.

**Comment by sampoder** [comment]
Also this document still doesn't feel right. IMO it should be made more professional

**Comment by nimit2801** [comment]
I'll reformat it, and update my PR!
@sampoder thanks for the review.

**Pull request closed without merging**